db/migrate/20150405012126_remove_approval_status_from_reservation.rb

```diff
@@ -0,0 +1,44 @@
+class RemoveApprovalStatusFromReservation < ActiveRecord::Migration
+  def change
+    Reservation.all.each do |res|
+      Reservation.transaction begin
```

Comment:
This should probably come before the `Reservation.all.each` loop so that we only proceed if **all** of the reservations save successfully, just to be safe, instead of each one (as it's currently set up)